src/app/state/store.tsx

```diff
@@ -17,26 +17,22 @@ export const middleware: Middleware[] = [
   userConsistencyCheckerMiddleware,
   notificationCheckerMiddleware,
 ];
-const defaultMiddlewareOptions = {
-  serializableCheck: process.env.NODE_ENV !== "test",
-};
-
 export const store = configureStore({
   reducer: rootReducer,
-  // [thunk, immutableStateInvariant, serializableStateInvariant] are all in the default middleware and included by default
-  // in development, with only thunk included in production.
-  // See https://redux-toolkit.js.org/api/getDefaultMiddleware#customizing-the-included-middleware
   middleware: (getDefaultMiddleware) => {
-    const newMiddleware = getDefaultMiddleware(defaultMiddlewareOptions).concat(middleware);
+    const newMiddleware = getDefaultMiddleware({
+      immutableCheck: false,
+      serializableCheck: false,
```

Comment:
Should this one be false also? I thought we were only concerned about the immutability check as that's the one being noisy?

Reply:
Yes, you're right! I will make changes to this on Monday morning!